datalad/support/annexrepo.py

```diff
@@ -335,3 +336,103 @@ def annex_add_to_git(self, files):
             self.cmd_call_wrapper.run(cmd_list, cwd=self.path)
         else:
             self.git_add(files)
+
+    def annex_initremote(self, name, options):
+        """Creates a new special remote
+
+        Parameters:
+        -----------
+        name: str
+            name of the special remote
```

Comment:
may be may be... I am still undecided though either AnnexRepo should be a GitRepo at all given all the direct-mode-ness ;)  
In the long shot I also see Dataset not being an AnnexRepo either -- may be we would become fancy to support various backends...? ;)